docs(portal): reserved paths (#8527)
Kong/docs.konghq.com: app/dev-portal/portals/customization/custom-pages.md

{:.note}
> *`home` is a special page representing the `/` path. If this page is deleted, you'll need to create it from the Pages API.*

#### Reserved paths

> The Portal requires a number of reserved paths from the root of the URL in order to properly function that may not be overridden with custom pages or other functionality.

Reviewer: ```suggestion
The Portal requires a number of reserved paths from the root of the URL to properly function. 
You cannot override these paths with custom pages or other functionality.
```

Author: Updated
